docs/usage/administration/hardening.md

```diff
@@ -0,0 +1,52 @@
+# Hardening 
+**Disclaimer: This guide is just a collection of ideas to make a synapse homeserver more secure - It is far from complete!**
+
+## General hardening
+- SSH hardening (key-based login only, change port, etc.)
+- Basic firewall (e.g. via `iptables`) - The following configuration is necessary for a matrix server to work properly (**This setup is not bulletproof, but should be a quite secure default!**):
+	- incoming https rule for basic server functionality, e.g.
+	`iptables -A INPUT -p tcp --dport 443 -m conntrack --ctstate NEW -j ACCEPT`
+	- outgoing rule for https (the server has to "speak" to other homeservers), e.g.
+	`iptables -A OUTPUT -p tcp --dport 443 -m conntrack --ctstate NEW -j ACCEPT`
+	- for also reaching homservers that were set up **without a reverse proxy** the outgoing port 8448 is needed (although there seems to be some uncertainty, whether this kind of setup is recommended, see https://github.com/matrix-org/synapse/issues/2438), e.g.
```

Comment:
> although there seems to be some uncertainty, whether this kind of setup is recommended, see https://github.com/matrix-org/synapse/issues/2438)

it is absolutely recommended to use a reverse proxy for your federation port, whether it be :8448 or :443. If I were to suggest one thing for "hardening" synapse, putting it behind a reverse proxy would be it.

#2438 is quite old.

Reply:
I added a sentence for using a reverse proxy and removed the old 'uncertainty' part.